src/tracks/playabletrack/wavetrack/ui/WaveTrackView.cpp

```diff
@@ -40,9 +40,7 @@ Paul Licameli split from TrackPanel.cpp
 #include "../../../../TrackInfo.h"
 
 #include "WaveTrackAffordanceControls.h"
-
-namespace {
```

Comment:
I would slightly prefer not removing the anonymous namespace over can, making symbol tables just a little smaller.  This would require forward-declaring incomplete types in namespace { ... } in the .h file.